### ISSUE TITLE
content_type_whitelist blocking image/svg+xml

### ISSUE THREAD

**Issue by sylvainbx** [carrierwaveuploader/carrierwave#2488]
I have an uploader with the following mime-type whitelist:
```
  def content_type_whitelist
    %w[image/svg+xml]
  end
``` 
When I upload an svg file, it fails with the following (crazy) error message:
```
{"project_cads.attachment":[
  "You are not allowed to upload image/svg+xml files, allowed types: image/svg+xml"
]}
```

Any idea on how to solve this?

**Comment by RigoTheDev** [comment]
Same issue. Seems like this line is rejecting `image/svg+xml` due to Regex https://github.com/carrierwaveuploader/carrierwave/blob/6c7b126bb4a2bc736d79467dc4a58fa559ac5696/lib/carrierwave/uploader/content_type_whitelist.rb#L43

**Comment by mshibuya** [comment]
Merged #2489.
